### ISSUE TITLE
Changing SMBIOS from 15,2 to 14,3 causes type-c DP out to stop working

### ISSUE THREAD

**Issue by themostunique** [cedric-bour/Acer-VN7-793g-Hackintosh#1]
I tried to change SMBIOS from MacBookPro 15,2 to MacBookPro 14,3 to more closely fit the model to the device. However, this caused type-c DP out to stop working.

**Comment by cedric-bour** [comment]
Have you test with EFI OR EFI_TEST_HDMI ?
Ans please test without AppleHDA.kext

**Comment by themostunique** [comment]
Before I change SMBIOS, type-c to DP works.
I have tested with EFI from the commit 8ce4d1c.
I would update and test EFI and EFI_TEST_HDMI, and without AppleHDA from the latest commit later.

**Comment by cedric-bour** [comment]
Okey good, maybe not change SMBIOS ahha.

Please test HDMI with last EFI, if not work remove AppleHDA.kext.
If not work test with EFI_TEST_HDMI, if not work remove AppleHDA.kext

EFI_TEST_HDMI more config in config.plist for HD 630

Please write link for your adaptater

Thanks

**Comment by themostunique** [comment]
HDMI does not work for EFI/EFI_TEST_HDMI and with/without AppleHDA.kext
I think this is because the HDMI is connected to GTX 1060 and we would need to downgrade to High Sierra for it to work.

Please write link for the guide you used to create the EFI. I want to try High Sierra for GTX 1060.

I think any type-c to displayport adapter will work, so you can buy from your local supplier/online store. I use a direct type-c to displayport cable: https://www.aliexpress.com/item/32989885494.html
and type-c to displayport adapter: https://www.aliexpress.com/item/32934863645.html
both works.

**Comment by cedric-bour** [comment]
I not have guide, i have build EFI
EFI work with Sierra, just disable in ACPI section all except GPU0

**Comment by themostunique** [comment]
Yes, but HDMI is directly connected to GTX 1060, so we can connect display just like a desktop computer.

**Comment by themostunique** [comment]
I would give High Sierra a try during the weekend

**Comment by cedric-bour** [comment]
When i plug USB c to HDMI, m'y pc crash, have you similar bug ? You plug before or after start pc ? Have you edit EFI file ?

**Comment by themostunique** [comment]
Let me try USB-C to HDMI. I currently use USB-C to DisplayPort and have not edit EFI or BIOS.

**Comment by cedric-bour** [comment]
Ok thanks and please test After and before start pc, with HDMI and DP

**Comment by themostunique** [comment]
I couldn't find my USB-C to HDMI adapter at the moment, but I can confirm that DP works when plugged in before and after starting pc.

**Comment by cedric-bour** [comment]
Mmh okey, i wait you find haha
And you have Catalina 10.15 ? Beta or final version ? And in System Information app, have you item in Thunderbolt section

**Comment by themostunique** [comment]
Catalina 10.15 Final.
In system information app, Thunderbolt section shows:
Thunderbolt:	No drivers are loaded.

**Comment by cedric-bour** [comment]
You have update to 10.15.1
Okey, and you have Acer VN7 793G mmh

**Comment by themostunique** [comment]
Yes 10.15.1, but Type-C to DP also works in 10.15.
Acer VN7 793G
i7-7700HQ
GTX 1060
16gb 2400mhz ddr4 ram

**Comment by cedric-bour** [comment]
Okey i have other version 
I have i5 and GTX 1050 Ti
I wait you find USB c to hdmi

**Comment by cedric-bour** [comment]
When your computer is started, you plug HDMI to your computer, crash or not ?

What is your version bios?

**Comment by cedric-bour** [comment]
When your computer is started, you plug HDMI to your computer, crash or not ?

After change SMBIOS, when your computer is started, you plug USB c to DP, crash ?

What is your version bios?

**Comment by themostunique** [comment]
Haven't tested HDMI.
After change SMBIOS, when my computer is started, I plug USB-C to DP, no crash but no signal.
My bios version is 1.11

**Comment by cedric-bour** [comment]
Okey.. please start pc and plug hdmi to default laptop, no adaptater, crash or no signal?

Thanks

**Comment by themostunique** [comment]
No Signal

**Comment by cedric-bour** [comment]
Upload and send me your EFI directory please

**Comment by cedric-bour** [comment]
Your internal graphics is 630 HD ?

**Comment by themostunique** [comment]
Yes

**Comment by themostunique** [comment]
[EFI.zip](https://github.com/cedric-bour/Acer-VN7-793g-Hackintosh/files/3796543/EFI.zip)
Here is my EFI/Clover folder

**Comment by cedric-bour** [comment]
Okey, you have other version of computer

I have update for graphic internal, please test "EFI_TEST_HDMI" and plug HDMI internal, not adaptater and tell me if you have signal 👍 
https://github.com/cedric-bour/Acer-VN7-793g-Hackintosh

**Comment by cedric-bour** [comment]
Okey

I have update EFI directory, please test plug HDMI (internal) and tell me if you have signal
